source/develop/release-management/features/network/lldp.html.md

```diff
@@ -500,7 +500,11 @@ The response in XML for the [SampleCaptures from Wireshark's wiki][5] will look
 
 ### Graphical User Interface
 
-Graphical User Interface is a [follow-up feature](#follow-up-features)
+LLDP information can be found in **Hosts** > **Network Interfaces** > 
+**Setup Host Networks**. Hovering over interface, e.g. `eth1`, will show tooltip
+with this information. UI filters values to display only important information.
```

Comment:
maybe:

Hovering over interface, e.g. `eth1`, will show tooltip with this information. UI filters values to display only important information. 
=>
The interface tooltip wll show the relevant information.

Reply:
Done